k8s/demo/redis/redis-cluster.yml

```diff
@@ -0,0 +1,101 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis-master
+  labels:
+    app: redis-master
+spec:
+  ports:
+    - port: 6379
```

Comment:
Since we are creating StatefulSets, add clusterIP: None and make this service a Headless Service.